package.json

```
@@ -4,7 +4,8 @@
   "license": "UNLICENSED",
   "scripts": {
     "lint": "eslint --ext js,jsx,ts,tsx .",
-    "format": "prettier --write \"**/*.{ts,tsx,js,jsx,json}\""
+    "format": "prettier --write \"**/*.{ts,tsx,js,jsx,json}\"",
+    "test": "cd ./node && yarn test"
```

I believe we can use the [`--cwd` arg from yarn](https://classic.yarnpkg.com/lang/en/docs/cli/#toc-cwd) instead here:

```suggestion
    "test": "yarn test --cwd ./node"
```